cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperConfiguration.java

```diff
@@ -7,29 +7,33 @@
 
 import io.strimzi.api.kafka.model.ZookeeperClusterSpec;
 
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
 
 /**
  * Class for handling Zookeeper configuration passed by the user
  */
 public class ZookeeperConfiguration extends AbstractConfiguration {
 
     private static final List<String> FORBIDDEN_OPTIONS;
-    private static final Map<String, String> DEFAULTS;
+    protected static final Map<String, String> DEFAULTS;
 
     static {
         FORBIDDEN_OPTIONS = asList(
                 ZookeeperClusterSpec.FORBIDDEN_PREFIXES.split(" *, *"));
 
-        DEFAULTS = new HashMap<>();
-        DEFAULTS.put("tickTime", "2000");
-        DEFAULTS.put("initLimit", "5");
-        DEFAULTS.put("syncLimit", "2");
-        DEFAULTS.put("autopurge.purgeInterval", "1");
+        Map<String, String> config = new HashMap<>();
+        config.put("tickTime", "2000");
+        config.put("initLimit", "5");
+        config.put("syncLimit", "2");
+        config.put("autopurge.purgeInterval", "1");
+        config.put("4lw.commands.whitelist", "*");
```

Comment:
Should this be added to the forbidden options list? Otherwise people can override it.

Reply:
Yes, good point. I will add it.

Reply:
I don't see this being done anywhere.